src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditMoney.razor

```diff
@@ -1,5 +1,5 @@
-﻿@using Cryptocash.Ui.Generated.Component;
-@inherits UiEditMoneyBase
+﻿@using MudBlazor
+@using static EditMoney
```

Comment:
I think 'using static' only brings benefits when EditMoney class has static members. In our case it seems to be redundant for me.

Reply:
Removed static and have change to Inherits latest commit - been forced to use UiEditMoney for class name instead as it clashes with razor class - cant seem to find a way around it - or would we prefer EditMoneyBase ? the class name I think needs to be different to the razor name